api/src/main/java/com/creatubbles/api/repository/AvatarRepositoryImpl.java

```diff
@@ -0,0 +1,45 @@
+package com.creatubbles.api.repository;
+
+import android.support.annotation.NonNull;
+
+import com.creatubbles.api.model.CreatubblesResponse;
+import com.creatubbles.api.model.user.avatar.Avatar;
+import com.creatubbles.api.model.user.avatar.AvatarSuggestion;
+import com.creatubbles.api.response.JsonApiResponseMapper;
+import com.creatubbles.api.response.ResponseCallback;
+import com.creatubbles.api.service.UserService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.jasminb.jsonapi.JSONAPIDocument;
+
+import java.util.List;
+
+import retrofit2.Call;
+
+/**
+ * Created by Janek on 26.10.2016.
+ */
+
+public class AvatarRepositoryImpl implements AvatarRepository {
```

Comment:
This class and it's constructor should be package private. The instance should only be created using builder.